tools/integration-test/src/integration-test-stack.test.ts

```diff
@@ -1,4 +1,3 @@
-import libraryInfo from '@guardian/cdk/lib/constants/library-info';
```

Comment:
Changes in this file are not related to this PR per se, they're made as they're flagged by the linter.